cmd/kube-scheduler/app/server.go

```diff
@@ -197,6 +197,7 @@ func Run(cc schedulerserverconfig.CompletedConfig, stopCh <-chan struct{}) error
 
 	// Prepare the event broadcaster.
 	if cc.Broadcaster != nil && cc.EventClient != nil {
+		cc.Broadcaster.StartLogging(klog.V(3).Infof)
```

Comment:
I don't have strong objection on this idea, but `V(3)` seems too much verbose

Reply:
I agree with @resouer. We should probably go with V(6).

Reply:
@bsalamat, Okay, I changed it to V(6), need some label, thanks.